linkml_runtime/utils/schemaview.py

```diff
@@ -104,6 +105,19 @@ def is_absolute_path(path: str) -> bool:
     drive, tail = os.path.splitdrive(norm_path)
     return bool(drive and tail)
 
+def resolve_import(sn: str, i: str) -> str:
```

Comment:
```suggestion
def _resolve_import(sn: str, i: str) -> str:
```

unless we expect this to be used elsewhere, probably make it private. it seems pretty specific to this one case in schemaview